dateparser/parser.py

```diff
@@ -281,7 +281,7 @@ def __init__(self, tokens, settings):
 
                 try:
                     microsecond = MICROSECOND.search(self.filtered_tokens[index + 1][0]).group()
-                    _is_after_time_token = token.index(":")
+                    token.index(":")  # raise ValueError if ':' can't be found
                     _is_after_period = self.tokens[self.tokens.index((token, 0)) + 1][0].index('.')
```

Comment:
`_is_after_period` seems unused as well.

Reply:
You are right, I pushed a new commit amending this.